infra/gcp/terraform/k8s-infra-prow-build/prow-build/resources/monitoring/grafana/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: monitoring
+  name: grafana
+  labels:
+    app: grafana
+spec:
+  type: ClusterIP
```

Comment:
@xmudrii Need to be `LoadBalancer`. 
I miss this.

Reply:
ACK, will create a PR to fix this.

Reply:
https://github.com/kubernetes/k8s.io/pull/6488